src/main/starlark/core/repositories/initialize.release.bzl

```diff
@@ -99,7 +99,6 @@ def kotlin_repositories(
         parent = KOTLIN_RULES,
         repo_mapping = {
             "@dev_io_bazel_rules_kotlin": "@%s" % KOTLIN_RULES.workspace_name,
-            "@": "@%s" % KOTLIN_RULES.workspace_name,
```

Comment:
@Wyverald how does Bazel surface this error in 6.0? We are using the 6.0 pre releases internally with a repository that's primarily Kotlin and haven't seen issues with this.

Reply:
It'll be an error that basically says that the empty string is not a valid repository name. This change just landed last week, so you might not have seen it yet; I'm only fixing this because rules_kotlin is currently red on our CI with Bazel@HEAD.